core/modules/layout/layout.module

```diff
@@ -536,6 +536,27 @@ function layout_page_access() {
   return call_user_func_array('_layout_page_access', $args);
 }
 
+/**
+ * Suppress use of layout.
+ *
+ * This function should be called from within another module's hook_init
+ * callback, when an empty layout (content only) should be displayed. This is
+ * useful for modules that implement popup pages or other special pages, where
+ * the default layout would be distracting, or just wasted space.
+ *
+ * @param $suppress
+ *   If set to TRUE, and called before menu_execute_active_handler(), the layout
+ *   will default to menu_default_route_handler().
+ *   The suppression state is returned.
+ */
+function layout_suppress($suppress = NULL) {
+  static $suppression_state = FALSE;
```

Comment:
Maybe someone can tell me: why "static"?

From above comment my understanding is that this runs *once* in hook_init(). Can there ever be a situation, where this code runs more than once per page call? IMO only then the static would make sense. But maybe I'm missing something.

Reply:
I think so it behaves like [backdrop_page_is_cacheable](https://docs.backdropcms.org/api/backdrop/core%21includes%21bootstrap.inc/function/backdrop_page_is_cacheable/1) as mentioned here https://github.com/backdrop/backdrop-issues/issues/1728#issuecomment-200631379

Reply:
I've no permission to do so, but consider this comment/question as outdated/resoved. :+1: